core/server/errors/no-database-found.js

```diff
@@ -0,0 +1,11 @@
+function NoDatabaseFound(message) {
+    this.message = message;
+    this.stack = new Error().stack;
+    this.statusCode = 404;
+    this.errorType = this.name;
+}
+
+NoDatabaseFound.prototype = Object.create(Error.prototype);
+NoDatabaseFound.prototype.name = 'NoDatabaseFound';
+
+module.exports = NoDatabaseFound;
```

Comment:
we will keep the specific error classes for now, because we need a way to differ certain situations.
see https://github.com/TryGhost/Ghost/pull/7019/files#diff-79a58308a1a36ab1b54544a17f88997aR87

in the near future we will optimise our error handling. see https://github.com/TryGhost/Ghost/issues/6526.
If we invent ghost error codes, we can get rid of specific error classes and check against error codes.

I only changed the wrong HTTP codes, which i totally misinterpreted and the name of the error class "DatabaseNotFound", because this error is thrown in case the database was never populated. 
see https://github.com/TryGhost/Ghost/pull/7019/commits/71ca05c1b95afcece1cb98fad0bd542a55b89423